WooCommerce/Classes/ViewRelated/Filters/FilterListViewController.swift

```diff
@@ -225,11 +220,11 @@ private extension FilterListViewController {
 
             switch selected.listSelectorConfig {
             case .staticOptions(let options):
-                self.cancellableSelectedFilterValue?.cancel()
+                self.selectedFilterValueSubscription?.cancel()
```

Comment:
Similar to the above comment, I think this `cancel()` method is redundant when we assign a new reference to the subscription right below.